src/wasm.h

```diff
@@ -796,6 +796,8 @@ class Expression {
   void dump();
 };
 
+static_assert(Expression::NumExpressionIds < 256, "ids must fit in a byte");
```

Comment:
The compiler should enforce how many values you have in your enum now that it has a forced type. https://godbolt.org/z/z1rn3cadP